vignettes/fims-demo.Rmd

```diff
@@ -351,35 +351,16 @@ population$SetRecruitment(recruitment$get_id())
 
 ## Putting It All Together 
 
-### Creating the FIMS Model and Making the TMB Function
-
+### Fitting the Model
 ```{r model}
 sucess <- CreateTMBModel()
```

Comment:
success instead of sucess?

Reply:
This is a problem in the package in general not of this PR, it is listed as a problem to fix in the quality control cleanup of M2 (e.g., Issue #662).